space-framework/entry.sh

```diff
@@ -47,6 +110,8 @@ echo "VERBOSITY=${VERBOSITY}"
 
 case "${ACTION}" in
 (space-manager)
+    set_kubeconfig_and_crds
+    set_provider_adapters
```

Comment:
Maybe the order should be? :
 set_kubeconfig_and_crds
run_space_manager
set_provider_adapters